server/utils/utils.js

```diff
@@ -42,6 +43,14 @@ const properCaseFullName = name =>
         .map(properCaseName)
         .join(' ')
 
+const stringToHash = inputString => {
+  return crypto.createHash('sha256').update(inputString).digest('base64')
```

Comment:
maybe force passing a salt here?

Reply:
or read the salt value from config here so the calling code doesn't need to pass it?

Reply:
ok. Have passed salt in a separate. Have also changed method name to isHashOfString as is more descriptive